E3Next/Processors/Rez.cs

```diff
@@ -701,7 +701,7 @@ private static void RegisterEvents()
                 {
                     string user = x.args[0];
 
-                    if (user == E3.CurrentName)
```

Comment:
forgot to do the spawn.DisplayName == E3.CurrentName types

Reply:
i just found all occurances of E3.CurrentName

Reply:
Also, have to remove the places where the first character is now upper cased as we are allowing anything to be in the config file and making it non case sensitive, so casting target will also have to use the new compairson i think

Reply:
I left spawn.DisplayName and CurrentName, since I wasn't sure if we wanted to keep those case comparisons. It sounds like everything can be case insensitive then? Or are there cases we care about case?

Reply:
have to go all the way wherever the name is referenced else a problem will happen.

Reply:
I can accept what we have so far here, but do you want to finish it?

Reply:
Yeah ill finish it this week

Reply:
I looked at target to see if we needed to add a .toLower() to when we add to _spawnsByName. However, when constructing the dictionary `public static Dictionary<string, Spawn> _spawnsByName = new Dictionary<string, Spawn>(2048, StringComparer.OrdinalIgnoreCase);` it's passed in a comparator that ignores case, so that looks like it should be fine?

Reply:
I don't understand? a non case sensitive dictionary only works on getting/setting values in that dictionary, not comparisons in if statements?

Reply:
other than using the containskey in said if staement

Reply:
I was just tracing usage of target for `so casting target will also have to use the new compairson i think`.

```
string target = E3.CurrentName
if (!_spawns.TryByName(target, out var spawn)) { }
```
So target should be fine, as getting/setting should be non case sensitive?

Reply:
basically anything with a == marks :P

Reply:
Oh, well then trybyname is fine :p. I was just trying to be exhaustive around other usages. Only thing I saw was the target = and Spell.CastTarget = CharacterName

![Screenshot 2024-03-19 095138](https://github.com/RekkasGit/E3Next/assets/1665885/fc0b3e69-fe6b-4272-bd24-b3af3ad8be7b)
![Screenshot 2024-03-19 095149](https://github.com/RekkasGit/E3Next/assets/1665885/71aec0bb-b155-4358-b37b-63c8535e93d0)